games.json

```diff
@@ -4557,13 +4557,17 @@
         "url": "https://www.shatterline.gg",
         "name": "Shatterline",
         "logo": "",
-        "status": "Broken",
+        "status": "Running",
         "native": false,
-        "reference": "https://www.protondb.com/app/2087030",
+        "reference": "https://github.com/ValveSoftware/Proton/issues/6165#issuecomment-1258731897",
         "anticheats": [
-            "Easy Anti-Cheat"
+            "Epic Online Services"
+        ],
+        "notes": [
+            [
+                "Requires windows release set on 7 (e.g. 'protontricks win7') to workaround the use of Winsock Input/Output (RIO) API which is not implemented in wine/proton"
```

Comment:
Make this shorter, in-depth Details are mostly uninteresting for most people

Reply:
I won't the details are important, merge as is or make your own version

Reply:
> I won't the details are important, merge as is or make your own version

You can add more in-depth details in the Updates section by linking to the relevant post that contains the information.  

Thanks for the PR, will adjust this on my own then